### PR TITLE
Feature transfers without quotes

### PR DESCRIPTION
**Pull request by vijayg10**
To enable sending transfers to scheme adapter without previous party lookup and quote request, had to remove some required fields in mojaloop-simulator incoming transfer request.

### CODE REVIEW

**Comment by millerabel** [comment]
Is this feature for perf testing?

-- Miller


> On Nov 22, 2019, at 5:48 PM, vijayg10 <notifications@github.com> wrote:
> 

﻿
To enable sending transfers to scheme adapter without previous party lookup and quote request, had to remove some required fields in mojaloop-simulator incoming transfer request.

You can view, comment on, or merge this pull request online at:

  https://github.com/mojaloop/mojaloop-simulator/pull/17

Commit Summary

Removed some required fields in transferRequest
Added build-perf in circleci yml file
File Changes

M .circleci/config.yml (36)
M src/simulator/api.yaml (5)
Patch Links:

https://github.com/mojaloop/mojaloop-simulator/pull/17.patch
https://github.com/mojaloop/mojaloop-simulator/pull/17.diff

**Comment by vijayg10** [comment]
Yes Miller, this is for perf testing.

**Comment by elnyry-sam-k** [comment]
@millerabel Yes - for perf testing only, so that we can baseline, because, thats what we did earlier during the PI4 (end of phase3) effort (so that we can compare apples to apples).

**Comment by millerabel** [comment]
I recall, yes, and makes sense. Thanks both!

Sent from my iPad

> On Nov 22, 2019, at 10:42 PM, Sam <notifications@github.com> wrote:
> 

﻿
@millerabel Yes - for perf testing only, so that we can baseline, because, thats what we did earlier during the PI4 (end of phase3) effort (so that we can compare apples to apples).

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,58 +24,60 @@ defaults_docker_helm_kube: &defaults_docker_helm_kube
...
-      git reset --hard "$CIRCLE_SHA1"
-      git checkout -q "$CIRCLE_TAG"
-    elif [ -n "$CIRCLE_BRANCH" ]
-    then
-      git reset --hard "$CIRCLE_SHA1"
-      git checkout -q -B "$CIRCLE_BRANCH"
-    fi
-
-    git reset --hard "$CIRCLE_SHA1"
+# Removed this because we are getting the following error in PR build process
```
Comment (mdebarros) [inline .circleci/config.yml]: I think you did not include the error or the explanation as to why this was commented out?

**Review by mdebarros — APPROVED** [review]
+1

**Review by mdebarros — CHANGES_REQUESTED** [review]
Have a look at  my comment.

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**